euphonic/spectra/base.py

```diff
@@ -976,29 +1021,38 @@ def copy(self: T) -> T:
                           copy.copy(self.x_tick_labels),
                           copy.deepcopy(self.metadata))
 
-    def get_bin_edges(self, bin_ax: Literal['x', 'y'] = 'x') -> Quantity:
+    def get_bin_edges(
+            self,
+            bin_ax: Literal['x', 'y'] = 'x',
+            *,
+            restrict_range: bool = True
+    ) -> Quantity:
         """
-        Get bin edges for the axis specified by bin_ax. If the size of
-        bin_ax is one element larger than z_data along that axis, bin_ax
-        is assumed to contain bin edges, but if bin_ax is the same size,
-        bin_ax is assumed to contain bin centres and a conversion is made.
-        In the conversion, the bin edges will not go outside the existing
-        data bounds so the first and last bins may be half-size. In addition,
-        each bin edge is assumed to be halfway between each bin centre,
-        which may not be an accurate assumption in the case of differently
-        sized bins.
+        Get bin edges for the axis specified by bin_ax. If the size of bin_ax
+        is one element larger than z_data along that axis, bin_ax is assumed to
+        contain bin edges. If bin_they are the same size, bin_ax is assumed to
```

Comment:
> if bin_they

? Looks like a train of thought got cut off.

Reply:
Good spot, should be
```suggestion
        contain bin edges. If they are the same size, bin_ax is assumed to
```